p2p/cmd/main.go

```diff
@@ -238,6 +238,18 @@ var (
 		},
 	})
 
+	optionValidatorRouterAddr = altsrc.NewStringFlag(&cli.StringFlag{
+		Name:    "validator-router-contract",
+		Usage:   "address of the validator router contract",
+		EnvVars: []string{"MEV_COMMIT_VALIDATOR_ROUTER_ADDR"},
+		Action: func(ctx *cli.Context, s string) error {
```

Comment:
Should we supply a default value for deployment on Holesky? If it exists

Reply:
We don't do it for oracle, so I didn't add default variable here (as it also comes with private key for the alchemy)